backend/src/database/state-migrations/migrations.ts

```diff
@@ -59,6 +60,7 @@ export const migrations: {
     8: treatmentSystemImprovements8,
     9: removeIsBeingTreated9,
     10: renameDeleteTransferAction10,
+    11: renameIncorrectImages11,
```

Comment:
```suggestion
    12: renameIncorrectPatientImages12,
```
(having 12 and maybe adding patient into the name or else make the name more clear, do we have a guideline for naming migrations?)

Reply:
Has been fixed (now also with the file-name)